CXX standard not allowed

```
-- Compiling with GCC
CMake Error at g2o/autodiff/CMakeLists.txt:2 (set_target_properties):
  INTERFACE_LIBRARY targets may only have whitelisted properties.  The
  property "CXX_STANDARD" is not allowed.


-- Generating position independent code for slam2d because Qt5 was built with -reduce-relocations
-- Configuring incomplete, errors occurred!
See also "/home/joris/codes/g2o/build/CMakeFiles/CMakeOutput.log".
See also "/home/joris/codes/g2o/build/CMakeFiles/CMakeError.log".
```

--- a/g2o/autodiff/CMakeLists.txt
+++ b/g2o/autodiff/CMakeLists.txt
@@ -1,7 +1,4 @@
 add_library(g2o_ceres_ad INTERFACE)
-set_target_properties(g2o_ceres_ad PROPERTIES
-    CXX_STANDARD 17
-)
 
 install(TARGETS g2o_ceres_ad
   EXPORT ${G2O_TARGETS_EXPORT_NAME}

